tasks/wait-for-service-broker/task

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+status="404"
+while [ $status == "404"  ]; do
+  status=$(curl -k -s -o /dev/null -w "%{http_code}" https://sample-odb.eight.loggr.cf-app.com/v2/catalog)
```

Comment:
This points directly to the eight environment. To use it across deployment we should probably take this in as a param